xep-0396.xml

```diff
@@ -34,6 +34,16 @@
   <registry/>
   <discuss>jingle</discuss>
   &paulschaub;
+  <revision>
+    <version>0.2.1</version>
```

Comment:
@vanitasvitae Adding the cipher specification (and, in a later revision, the Registry entry) is not purely editorial, so you need to bump the version to 0.3.0 at least.

Reply:
Alright :)